tests/utils.go

```diff
@@ -321,137 +343,141 @@ func resizeAndVerifyPVC(shouldPass bool, size string) {
 }
 func createDeployVerifyApp() {
 	ginkgo.By("creating and deploying app pod")
-	createAndDeployAppPod(appName)
+	createAndDeployAppPod(appNames)
 	time.Sleep(30 * time.Second)
 	ginkgo.By("verifying app pod is running", verifyAppPodRunning)
```

Comment:
This log line seems to suggest that it's checking for one app Pod's status. Maybe this would be better?
```suggestion
	ginkgo.By("verifying app pods are running", verifyAppPodRunning)
```

Reply:
Updated